sdk/appconfiguration/app-configuration/rollup.base.config.js

```diff
@@ -23,8 +25,27 @@ const banner = [
   " */"
 ].join("\n");
 
+const ignoreKnownWarnings = (warning) => {
+  if (warning.code === "THIS_IS_UNDEFINED") {
+    // This error happens frequently due to TypeScript emitting `this` at the
+    // top-level of a module. In this case its fine if it gets rewritten to
+    // undefined, so ignore this error.
+    return;
+  }
+
+  if (
+    warning.code === "CIRCULAR_DEPENDENCY" &&
+    warning.importer.indexOf(path.normalize("node_modules/chai/lib") === 0)
+  ) {
+    // Chai contains circular references, but they are not fatal and can be ignored.
+    return;
+  }
+
+  console.error(`(!) ${warning.message}`);
```

Comment:
Weird.. I don't see the above warnings when building app config locally from master branch without these changes...

Reply:
Removed!

Reply:
Brought it back - when I started stripping out some of the shimming I was doing the warning came back.